dev.zed.Zed.yaml

```diff
@@ -0,0 +1,135 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
```

Comment:
I can understand comments may seem excessive, but the intent was to have context in one place so other readers can understand why it was being added if they wanted to understand why the manifest was made as it is.

In another way, I also did this for myself since I needed to understand why one permission would be accepted or not since it felt like standards evolved since my last Flatpak submission. For example, I was never exposed to the [linter exception process before](https://github.com/flathub/flathub/pull/5253#issuecomment-2118607038) and it felt useful to let others know they need to ask for it when they want to add that permission by default.

Finally, it was also intended to help answer questions from the PR template such as ` I am using only the minimal set of permissions. (If not, please explain each non-standard permission.)`